mittab/apps/tab/models.py

```diff
@@ -296,6 +315,71 @@ class Meta:
         ordering = ["name"]
 
 
+class Outround(models.Model):
+    VARSITY = 0
+    NOVICE = 1
+    TYPE_OF_ROUND_CHOICES = (
+        (VARSITY, "Varsity"),
+        (NOVICE, "Novice")
+    )
+
+    num_teams = models.IntegerField()
+    type_of_round = models.IntegerField(default=VARSITY,
+                                        choices=TYPE_OF_ROUND_CHOICES)
+    gov_team = models.ForeignKey(Team, related_name="gov_team_outround",
+                                 on_delete=models.CASCADE)
+    opp_team = models.ForeignKey(Team, related_name="opp_team_outround",
+                                 on_delete=models.CASCADE)
+    chair = models.ForeignKey(Judge,
+                              null=True,
+                              blank=True,
+                              on_delete=models.CASCADE,
+                              related_name="chair_outround")
+    judges = models.ManyToManyField(Judge, blank=True, related_name="judges_outrounds")
+    UNKNOWN = 0
+    GOV = 1
+    OPP = 2
+    GOV_VIA_FORFEIT = 3
+    OPP_VIA_FORFEIT = 4
+    VICTOR_CHOICES = (
+        (UNKNOWN, "UNKNOWN"),
+        (GOV, "GOV"),
+        (OPP, "OPP"),
+        (GOV_VIA_FORFEIT, "GOV via Forfeit"),
+        (OPP_VIA_FORFEIT, "OPP via Forfeit"),
+    )
+    room = models.ForeignKey(Room,
+                             on_delete=models.CASCADE,
+                             related_name="rooms_outrounds")
+    victor = models.IntegerField(choices=VICTOR_CHOICES, default=0)
+
+    def clean(self):
+        if self.pk and self.chair not in self.judges.all():
+            raise ValidationError("Chair must be a judge in the round")
+
+    def __str__(self):
+        return "Outround {} between {} and {}".format(self.num_teams,
+                                                      self.gov_team,
+                                                      self.opp_team)
+
+    @property
+    def winner(self):
+        if self.victor in [1, 3]:
```

Comment:
nit: use the constants here to make it more readable

```suggestion
        if self.victor in [self.GOV, self.GOV_VIA_FORFEIT]:
```